sample-lnd.conf

```diff
@@ -1501,3 +1501,21 @@ litecoin.node=ltcd
 ; are sent over a short period.
 ; htlcswitch.mailboxdeliverytimeout=60s
 
+[grpc]
+
+; How long the server waits on a gRPC stream with no inactivity before pinging
```

Comment:
My understanding of grpc.server-ping-time (GRPC_ARG_KEEPALIVE_TIME_MS) is, the amount of time that the connection must be idle before a keepalive probe (ping) is sent.

Getting confused by the wording `no inactivity` here. Shouldn't it be `no activity` or just `inactivity` instead?

Reply:
I think you are right, that double negative is likely a mistake.

Reply:
Ah yes, my bad. Fixed.